qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -1232,7 +1232,7 @@ class MCXSynthesis2DirtyKG24(HighLevelSynthesisPlugin):
 
     For a multi-controlled X gate with :math:`k\ge 3` control qubits this synthesis method requires
     :math:`2` additional dirty ancillary qubits. The synthesized circuit consists of :math:`k + 2`
-    qubits and at most :math:`24 * k - 48` CX gates.
+    qubits and at most :math:`12 * k - 18` CX gates.
 
```

Comment:
The synthesized circuit consists of k + 3 qubits, right? (k controls, 1 target and 2 ancillas)